src/Workspaces/Remote/ServiceHub/Services/CodeAnalysisService_SymbolFinder.cs

```diff
@@ -168,10 +168,10 @@ public FindLiteralReferencesProgressCallback(CodeAnalysisService service)
             }
 
             public Task ReportProgressAsync(int current, int maximum)
-                => _service.Rpc.InvokeAsync(nameof(ReportProgressAsync), current, maximum);
+                => _service.InvokeAsync(nameof(ReportProgressAsync), new object[] { current, maximum }, CancellationToken.None);
```

Comment:
why CancellationToken.None?

Reply:
ReportProgressAsync doesn't give in cancellation.

Reply:
The FindLiteralReferencesProgressCallback should now be passed the appropriate cancellation token to control this.

Reply:
IIRC, at one point these didn't take cancellation tokens, and couldn't have them passed along since they would interfere with the params-object[].  If these can now take cancellation tokens, we should just pass through the right values.